src/main/proto/network.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+//option java_multiple_files = true;
+package core.src.edu.rice.comp413fall2020grey.Networking;
+
+
+// Request payload
+message ServerInformation {
+  // Each message attribute is strongly typed.
+  // You also must assign a "tag" number.
+  // Each tag number is unique within the message.
+  string server_name = 1;
+
+  // This defines a strongly typed list of String
+  string server_type = 2;
+
+  // There are many more basics types, like Enum, Map
+  // See https://developers.google.com/protocol-buffers/docs/proto3
+  // for more information.
+}
+
+message ConnectionConfirmation {
+  string confirmation = 1;
+}
+
+// Defining a Service, a Service can have multiple RPC operations
+service ConnectionService {
+  // Define a RPC operation
+  rpc connectTo(ServerInformation) returns (ConnectionConfirmation);
```

Comment:
Shall we merge the messages and service defined in `network.proto` into `grpc.proto`?

Reply:
No, I think we can remake those anyways. the network.proto stuff is just there to serve as an example

Reply:
Then may I remove `rpc connectTo(ServerInformation) returns (ConnectionConfirmation);` in `rbox.proto`? It's giving me error `"ServerInformation" is not defined. "ConnectionConfirmation" is not defined.` when I tried to compile.

Reply:
Yeah, you can delete all the network.proto stuff from rbox.proto